front/app/components/UI/Button/index.tsx

```diff
@@ -16,6 +16,7 @@ interface Props extends ButtonProps {
 
 interface ButtonContainerProps extends ComponentLibraryButtonContainerProps {
   'data-testid'?: string;
+  'data-cy'?: string;
```

Comment:
Given that our codebase is almost entirely made up of generated classnames, I can't really think of a better way to do it than give specific, low-level components (like a button) an explicit ID for testing purposes